Reference/Searching/Examine/quick-start.md

```diff
@@ -55,74 +56,54 @@ In MVC we have a method called `TypedSearch` on the `UmbracoHelper` which will r
                 </li>
             }
         </ul>
-    }   
+    }
 
-### Razor
+### ExamineManager
 
-In razor macros there's a `Search` method on the `DynamicNode` model which will return a `DynamicNodeList`:
+To complete more complex searches, we can use the ExamineManager and our custom search provider to expose further functionality, such as being able to handle mispellings, for example.
 
```

Comment:
I would keep a simple razor example first before diving deep and using the examine manager.  
We could move this (good) example to the examine manager page. And maybe link (again) to that pace from this section.

    foreach (var result in Umbraco.TypedSearch(query)) {
        <li>@resul...</li>
    }